File: ClientApp/src/types/practices/responses/ResponsePractice.ts
export interface ResponsePractice {
    id_practice: string;
    id_student: string;
    id_company: string;
    academic_year: string;
    study: string;
    begin_date: Date;
    end_date: Date | null;
    position: string | null;
    hours_worked: number | null;
    mentor: string | null;
    mentor_comment: string | null;
    job_description_practice_diary: string | null;
}

File: Models/Practices/Responses/ResponsePractice.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Models.Practices.Responses
{
    public class ResponsePractice
    {
        public string Id_practice { get; set; } = null!;
        public string Id_student { get; set; } = null!;
        public string Id_company { get; set; } = null!;
        public string Academic_year { get; set; } = null!;
        public string Study { get; set; } = null!;
        public DateTime Begin_date { get; set; }
        public DateTime? End_date { get; set; }
        public string? Position { get; set; }
        public int? Hours_worked { get; set; }
        public string? Mentor { get; set; }
        public string? Mentor_comment { get; set; }
        public string? Job_description_practice_diary { get; set; }
    }
}
